teacher.rb

```diff
@@ -0,0 +1,17 @@
+require './person'
+
+# The Teacher class represents a teacher
+# in a school. The teacher has a name, age,
+# and specialization. The teacher inherits
+# from the Person class.
+# The teacher can use the services that
+class Teacher < Person
+  def initialize(name, age, specialization)
+    super(name, age)
```

Comment:
- [ ] please make sure to also add the parent_permission in the initialize method and pass it to super as well.

Reply:
Resolved as advised